lib/sensu/api/routes/silenced.rb

```diff
@@ -3,8 +3,8 @@ module API
     module Routes
       module Silenced
         SILENCED_URI = /^\/silenced$/
-        SILENCED_SUBSCRIPTION_URI = /^\/silenced\/subscriptions\/([\w\.-:]+)$/
-        SILENCED_CHECK_URI = /^\/silenced\/checks\/([\w\.-]+)$/
+        SILENCED_SUBSCRIPTION_URI = /^\/silenced\/subscriptions\/([\w\.\-:]+)$/
+        SILENCED_CHECK_URI = /^\/silenced\/checks\/([\w\.\-]+)$/
```

Comment:
Even without changing SILENCED_CHECK_URI I'm curious how the implementation here compares with https://github.com/sensu/sensu-settings/blob/master/lib/sensu/settings/validators/check.rb#L12